test/config-luatest/instance_config_schema_test.lua

```diff
@@ -967,3 +967,25 @@ g.test_app = function()
     local res = instance_config:apply_default({}).app
     t.assert_equals(res, nil)
 end
+
+g.test_security = function()
```

Comment:
I'm looking into PR #8895 as well and there are two good ideas in it:

* Add a test that starts a real server and verifies `box.cfg.<...>` values (see config_test.lua).
* Add enterprise/community `:validate()` and `:apply_default()` test cases both. The community one would have different defaults and would receive an error on attempt to set some options/values.

Reply:
> Add a test that starts a real server and verifies box.cfg.<...> values (see config_test.lua).

If you intend to request such test on every section, it needs some gentrification. Copy-pasting minimal config every time is meaningless.

Reply:
> Add enterprise/community :validate() and :apply_default() test cases both. The community one would have different defaults and would receive an error on attempt to set some options/values.

We already do test the defaults in `cluster_config_schema_test.lua`, both EE and CE. Added test to check validation error in CE.

Reply:
It is my personal feeling that without a kind of end-to-end scenario in the automatic testing we can broke some options without noticing it. This is a weak requirement, but it is a ground level I would require from these patches.

I looked over the patch and now the testing looks quite good. Thanks!